### ISSUE TITLE
Extension doesn't work with UNC paths (Samba directory)

### ISSUE THREAD

**Issue by FezVrasta** [zaggino/brackets-git#216]
Not sure if it's a problem of Brackets or of Brackets-git but if I open a repository stored in a Samba directory, the .git folder is not shown in the file-tree and brackets-git is loaded like my folder is not a git-root.

Using command line everything works fine.

**Comment by stevenmg** [comment]
Not sure what is up, but I never experienced this issue until today. I did update the extension last week which may have broken it. Any chance this had been working but was broken again in the latest version?

Also, one way to accomplish this would be with #1072

**Comment by zaggino** [comment]
I have no knowledge of what would break it with the update. Can you try getting back to old version? You can browse any commit on github and download a zip to install with drag & drop to extension manager.

**Comment by stevenmg** [comment]
I reverted to 0.16.0 and it works again.

**Comment by zaggino** [comment]
Does `0.16.5` work too?

**Comment by stevenmg** [comment]
Yes it does. I checked all commits between 0.16.5 and 0.16.6, and I found one commit that looked interesting: 6ac083f4efffb7c727187d993fb064931189fecb

Not sure what cross spawn is, but this is the only thing I found that does anything related to command handling.
